src/components/m-dropdown/index.js

```diff
@@ -27,9 +28,11 @@ class AXADropdown extends BaseComponentGlobal {
     super.render();
 
     const inFlow = this.hasAttribute('in-flow');
+    const size = getAttribute('size');
 
     this.className = classnames(this.initialClassName, 'm-dropdown js-dropdown', {
       'm-dropdown--in-flow': inFlow,
+      'm-dropdown--small': !!size,
```

Comment:
Won't we use a more consistent syntax, because size isn't a boolean here, like in the `m-button`:
```js
{
  [`m-button--${size}`]: size,
}
```

Reply:
so to have instead of `--small` `--sm`? yes makes sense i will change it

Reply:
yep

Reply:
i like it.. much better agree